plugins/services/src/js/components/modals/ServiceRootGroupModal/utils.tsx

```diff
@@ -57,17 +59,34 @@ export function getPathFromGroupId(id: string): string {
   return `/${id}`;
 }
 
+function addFormError(
+  errors: GroupFormErrors,
+  formError: JSX.Element
+): GroupFormErrors {
+  if (!errors.form) {
```

Comment:
well we treat `errors = {}` as no errors present. so that is why we don't initialize `errors.form` until we have an actual error. Maybe this could be refactored.